CMakeLists.txt

```diff
@@ -253,7 +253,8 @@ add_library(devilution STATIC
   Source/town.cpp
   Source/towners.cpp
   Source/track.cpp
-  Source/trigs.cpp)
+  Source/trigs.cpp
+  Source/Misc/config.cpp)

```

Comment:
creating a new file just for 1 function seems to be the very definition of an overkill

Reply:
The idea is to use this moving forward for more config options rather than scattering them throughout the code. And considering the source files in Diablo are absolutely enormous, I personally don't think it's a bad idea to start adding more source files rather than keeping adding more and more to the existing ones.

But I can remove this file and move the config check somewhere else.

Reply:
New files should go in SourceX, shared headers got in SourceS